src/NzbDrone.Core/Notifications/Webhook/WebhookRelease.cs

```diff
@@ -31,5 +33,6 @@ public WebhookRelease(QualityModel quality, RemoteEpisode remoteEpisode)
         public long Size { get; set; }
         public int CustomFormatScore { get; set; }
         public List<string> CustomFormats { get; set; }
+        public List<Language> Languages { get; set; }
```

Comment:
Just making sure, but this serializes the whole Language object right and not just the ID like we have when storing in the DB?

Reply:
yep, it is a list of Language objects, so `id` and `name`

eg:

```json

{
  "series": {
    ...
    "originalLanguage": {
      "id": 1,
      "name": "English"
    }
  },
   ...
  "episodeFile": {
   ...
    "languages": [
      {
        "id": 1,
        "name": "English"
      }
    ],
   ...
  ],
  "customFormatInfo": {
    "customFormats": [],
    "customFormatScore": 0
  },
...
}
```